### PR TITLE
Breadcrumbs work with http errors

### PR DESCRIPTION
**Pull request by Yrob**
If an instance view returns a 404 or other http error
this will now display nicely in the page title and breadcrumbs

### CODE REVIEW

File: src/rest_framework_dso/views.py [hunk 1]
```diff
@@ -41,12 +41,15 @@ def get_view_name(view):
...
         name = getattr(view.schema_generator, "title", "Schema")
 
-    # Add the title for instances
+    # Use the title for instances
     response = getattr(view, "response", None)
     suffix = getattr(view, "suffix", None)
 
     if suffix == "Instance" and response:
-        name = response.data["_links"]["self"]["title"]
+        if response.status_code == 200:
```
Comment (jjmurre) [inline src/rest_framework_dso/views.py:49]: Nice solution!

**Review by jjmurre — APPROVED** [review]
LGTM

File: src/rest_framework_dso/views.py (hunk 1 above)
Comment (Yrob) [reply src/rest_framework_dso/views.py:49]: thanks

Comment (ngaranko) [reply src/rest_framework_dso/views.py:49]: Please use `rest_framework.status. HTTP_200_OK`. Also adding test is good idea.

**Pull request merged**